### DISCUSSION TITLE
5.8.0

### DISCUSSION THREAD

**Discussion by kobylynskyi** [kobylynskyi/graphql-java-codegen#1273] [🚀 Releases]
## What's Changed
* Ability to supply a root directory with custom templates: `customTemplatesRoot` #1158 #1198
* Increase customizability of values that can be returned by `apiReturnType`/`apiReturnListType` via `{{TYPE}}` placeholder #1167 #1200
* Enhance Schema Finder to follow symbolic links #1199
* Relay support for client classes #1202
* Kotlin: fix annotations placement on type and input classes #1264
 
## New Contributors
* @isaac-mercieca made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1198
* @matsudamper made their first contribution in https://github.com/kobylynskyi/graphql-java-codegen/pull/1201

**Full Changelog**: https://github.com/kobylynskyi/graphql-java-codegen/compare/v5.7.2...v5.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kobylynskyi/graphql-java-codegen/releases/tag/v5.8.0'>5.8.0</a>.</em>

**Comment by kobylynskyi** [comment]
@jxnu-liguobin please release SBT plugin.

**Reply by jxnu-liguobin** [reply]
When we released 6.0.0, we modify the SBT plugin to a unified OSS repo? What do you think?
